### DISCUSSION TITLE
How should I profile my configuration?

### DISCUSSION THREAD

**Discussion by otakutyrant** [nushell/nushell#13584] [Q&A] [answered]
Suddenly, my nushell often stucks! It even takes four seconds to change working directory.

I tried `debug profile { source $nu.config-path }`, but it complained as below:

```
Error: nu::parser::extra_positional

  × Extra positional argument.
   ╭─[entry #9:1:1]
 1 │ debug profile { source $nu.config-path }
   ·       ───┬───
   ·          ╰── extra positional argument
   ╰────
  help: Usage: debug {flags}
```

**Comment by fdncred** [comment, marked as answer]
It looks like you may have an old version of nushell because the error with debug profile is looking like it doesn't exist.

**Reply by otakutyrant** [reply]
My bad. I forgot that I hadn't upgrade nushell for a long time. Because it often breaks my config compatibility, it is painful!
